kibana_saved_objects/kibana_visualizations.ndjson

```diff
@@ -0,0 +1,137 @@
+{"attributes":{"description":"","kibanaSavedObjectMeta":{"searchSourceJSON":"{\n  \"filter\": [],\n  \"query\": {\n    \"query\": \"response:200 && probably_a_bot:false\",\n    \"language\": \"lucene\"\n  },\n  \"indexRefName\": \"kibanaSavedObjectMeta.searchSourceJSON.index\"\n}"},"title":"ReactomeWS - Top 10 Cystoscape download files","uiStateJSON":"{\n  \"vis\": {\n    \"params\": {\n      \"sort\": {\n        \"columnIndex\": null,\n        \"direction\": null\n      }\n    }\n  }\n}","version":1,"visState":"{\n  \"title\": \"ReactomeWS - Top 10 Cystoscape download files\",\n  \"type\": \"table\",\n  \"params\": {\n    \"perPage\": 10,\n    \"showPartialRows\": false,\n    \"showMeticsAtAllLevels\": false,\n    \"sort\": {\n      \"columnIndex\": null,\n      \"direction\": null\n    },\n    \"showTotal\": false,\n    \"totalFunc\": \"sum\"\n  },\n  \"aggs\": [\n    {\n      \"id\": \"1\",\n      \"enabled\": true,\n      \"type\": \"count\",\n      \"schema\": \"metric\",\n      \"params\": {}\n    },\n    {\n      \"id\": \"2\",\n      \"enabled\": true,\n      \"type\": \"terms\",\n      \"schema\": \"bucket\",\n      \"params\": {\n        \"field\": \"reactome_cytoscape_download_file_name.raw\",\n        \"size\": 10,\n        \"order\": \"desc\",\n        \"orderBy\": \"1\",\n        \"customLabel\": \"Download file name (via Cytoscape)\"\n      }\n    }\n  ]\n}"},"id":"000e7440-1b24-11e8-b589-376955287594","migrationVersion":{"visualization":"7.3.1"},"references":[{"id":"REACTOMEWS_INDEX","name":"kibanaSavedObjectMeta.searchSourceJSON.index","type":"index-pattern"}],"type":"visualization","updated_at":"2019-12-12T22:17:04.425Z","version":"WzUwMiwxMV0="}
```

Comment:
Where do all these `ndjson` files come from?

Reply:
They are exported from Kibana. If you want to preserve your Saved Objects (Visualisations, Dashboards, Searches, etc...) you need to save them to a file. Kibana will save them in ndjson files. Ndjson files are just json files with a full JSON object on each line (http://ndjson.org/). This means the lines can get very wiiiiiiiiiiiiiiiiiiiiiiiiiiiiide. I don't think these files are meant to be (easily) human-readable, but I want to back these Saved Objects up _somewhere_.

Reply:
Got it, was just curious since I doubted you had wrote them up yourself :P